istioctl/cmd/workload.go

```diff
@@ -333,6 +335,29 @@ func createCertsTokens(kubeClient kube.ExtendedClient, wg *clientv1alpha3.Worklo
 	}
 
 	serviceAccount := wg.Spec.Template.ServiceAccount
+	tokenPath := filepath.Join(dir, "istio-token")
+	jwtPolicy, err := util.DetectSupportedJWTPolicy(kubeClient.RESTConfig())
+	if err != nil {
+		fmt.Printf("Failed to determine JWT policy support: %v", err)
+	}
+	if jwtPolicy == util.FirstPartyJWT {
+		fmt.Println("warning: Detected that your cluster does not support third party JWT authentication. " +
+			"Falling back to less secure first party JWT. " +
+			"See " + url.ConfigureSAToken + " for details.")
+		sa, err := kubeClient.CoreV1().ServiceAccounts(wg.Namespace).Get(context.TODO(), serviceAccount, metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+		secret, err := kubeClient.CoreV1().Secrets(wg.Namespace).Get(context.TODO(), sa.Secrets[0].Name, metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+		if err := ioutil.WriteFile(tokenPath, secret.Data["token"], filePerms); err != nil {
+			return err
+		}
+		fmt.Printf("warning: a security token for namespace %s and service account %s has been generated and stored at %s\n", wg.Namespace, serviceAccount, tokenPath)
```

Comment:
```suggestion
		fmt.Printf("Warning: a security token for namespace %q and service account %q has been generated and stored at %s\n", wg.Namespace, serviceAccount, tokenPath)
```

I realize you were following the existing error message.  Either change both or ignore my suggestion.

Reply:
changed both